lumigator/backend/backend/services/jobs.py

```diff
@@ -381,6 +381,9 @@ def create_job(
             job_type = JobType.INFERENCE
             if not request.output_field:
                 request.output_field = "predictions"
+
+            if request.task == "text-generation" and not request.system_prompt:
+                raise ValueError("System prompt is required for text generation tasks.")
```

Comment:
Hey @dpoulopoulos , could you create a `JobValidationError` in `job_exceptions.py` and raise that instead?

It can then be explicitly documented as a type of error raised by the service, and you could add a mapping in `job_exception_mappings` for the `400` too.

Reply:
@peteski22 @javiermtorres can you check if it's okay now? I tested it locally and it seems it raises the right error.

> None: This code is now outdated.